monai/tests/test_clinical_preprocessing.py

```
@@ -0,0 +1,106 @@
+    dummy_data = np.random.randn(64, 64, 64).astype(np.float32)
+    test_file = tmp_path / "test.nii.gz"
+
+    write_nifti(dummy_data, test_file)
+
+    # Test with each modality
+    for modality in ["CT", "MRI"]:
+        try:
+            result = preprocess_dicom_series(str(test_file), modality)
+            assert result is not None
+            assert hasattr(result, "shape")
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)